src/lib/main.js

```diff
@@ -194,6 +194,25 @@ module.exports = async (argv, env) => {
     await checkForUpdates({ nonFatal: true });
   }
 
+  let validationsPassed = true;
+  info('Starting actions validations…');
+  for (let action of actions) {
```

Comment:
This is running all of the validations before any execute happens.  In cases where we are developing forms for the first time or the xml doesn't exist or the xlsx was modified and a new xml is generated. The validation won't occur again after the `convert-X-forms` runs which could cause the original problem this is attempting to solve.  I believe the the validation should always follow the execute.

Reply:
Yeah you're right. Good catch! The validation here needs to happen after the `convert` step.